frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -211,56 +296,24 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
       <Box flex="1">
         <Tabs marginTop={3}>
           <TabList>
-            <Tab onClick={() => setSearch("")} color={unselectedColor}>
+            <Tab onClick={() => setStatus("")} color={unselectedColor}>
               All
             </Tab>
-            <Tab onClick={() => setSearch("Draft")} color={unselectedColor}>
+            <Tab onClick={() => setStatus("DRAFT")} color={unselectedColor}>
               Drafts
             </Tab>
-            <Tab onClick={() => setSearch("Published")} color={unselectedColor}>
+            <Tab onClick={() => setStatus("PUBLISHED")} color={unselectedColor}>
               Published
             </Tab>
-            <Tab onClick={() => setSearch("Archived")} color={unselectedColor}>
+            <Tab onClick={() => setStatus("ARCHIVED")} color={unselectedColor}>
               Archived
             </Tab>
           </TabList>
           <TabPanels>
-            <TabPanel>
-              <VStack pt={4} spacing={6}>
-                <HStack width="100%">
-                  <SearchBar onSearch={setSearch} />
-                  <SortMenu
-                    properties={[
-                      "status",
-                      "name",
-                      "grade",
-                      "type",
-                      "country",
-                      "region",
-                    ]}
-                    onSortOrder={setSortOrder}
-                    onSortProperty={setSortProperty}
-                  />
-                  <FilterMenu />
-                </HStack>
-                {search && (
-                  <Text fontSize="16px" color="grey.300" width="100%">
-                    Showing {sampleAssessments.length} results for &quot;
-                    {search}&quot;
-                  </Text>
-                )}
-                <AssessmentsTable assessments={assessments} />
-              </VStack>
-            </TabPanel>
-            <TabPanel>
-              <AssessmentsTable assessments={assessments} />
-            </TabPanel>
-            <TabPanel>
-              <AssessmentsTable assessments={assessments} />
-            </TabPanel>
-            <TabPanel>
-              <AssessmentsTable assessments={assessments} />
-            </TabPanel>
+            {FullTabPanel}
+            {FullTabPanel}
+            {FullTabPanel}
+            {FullTabPanel}
```

Comment:
?

Reply:
Each tab uses the same Tab Panel, so rather then, as I say that I realized i could perhaps just map it to a single constant wait let me fix that